handbook/product/product_management/direction_template.md

```diff
@@ -0,0 +1,101 @@
+# Vision, Strategy, and Roadmap Template
+
+> ℹ️  The following is a template for use on the pages of specific product & engineering teams, used to record and share the vision, strategy, and roadmap.<br>Copy to your team's page and complete accordingly.
+
+> As you write this document, keep in mind that focusing on problems to solve more than lists of features tends to be a better way to approach communicating with customers, as it leaves room for learning between what you know now and what you deliver in the end. This is not to say you never talk about features, but especially as you get [further out in planning](https://medium.com/@jobv/the-nearsighted-roadmap-57fa57b5906a) the blend of features vs. problems to solve should move more towards the latter.
+
+> Be sure also to explain [why something is part of your direction](https://medium.com/leadership-motivation-and-impact/the-power-of-starting-with-why-f8e491392ef8). What you are doing is interesting, but why you have chosen that of all the possibilities is often even more enlightening.
+
+---
+
+# MyProductArea Direction
+
+This page outlines the vision, strategy and goals of the MyProductArea team. You can use this section to describe what your scope is, and where your product is today.
+
+Quicklinks:
+
+- [MyProductArea Roadmap](https://sourcegraph.productboard.com/roadmap/) <!-- Update links with your page -->
```

Comment:
These were no longer updated in Q2, in Q3 we moved to one-pagers which will be integrated here.

```suggestion
```